apps/bfd-server/bfd-server-war/pom.xml

```diff
@@ -14,6 +14,17 @@
         The primary WAR for the BFD Server project. Provides a FHIR API.
     </description>
 
+    <repositories>
+        <repository>
+            <id>spring-releases</id>
+            <name>Spring Releases</name>
+            <url>https://repo.spring.io/release</url>
+            <snapshots>
+                <enabled>false</enabled>
+            </snapshots>
+        </repository>
+    </repositories>
+
```

Comment:
I honestly don't know what to do here because the `spring-retry` library is _supposed_ to be available on Maven Central, but it seems that [just isn't the case](https://repo.maven.apache.org/maven2/springframework/). However, the Spring Release repository does seem to have the latest release version of `spring-retry` available.